app/views/workspaces/_form.html.erb

```diff
@@ -9,7 +15,7 @@
         <%= f.input :daily_rate, placeholder: '$14.50', input_html: { min: '1', step: '1' } %>
         <%= f.input :photo, as: :file %>
         <div class=" h-centered">
-        <%= f.button :submit, "Create", class: "btn btn-home u-margin-bottom-md" %>
+        <%= f.button :submit, "Update", class: "btn btn-edit u-margin-bottom-md" %>
```

Comment:
est-ce que ça ne le change pas aussi dans le form NEW étant donné que c'est une partial?

Reply:
Bonne question, il faut que je check

Reply:
I'll change to submit

Reply:
Done